practice-app/website/api/doc/discussions_POST.yml

```
@@ -0,0 +1,42 @@
+Endpoint to create a discussion post
+---
+parameters:
+  - name: body
+    in: body
+    required: true
+    schema:
+      type: object
+      required:
+        - event_id
+        - text
+      properties:
+        event_id:
+          type: integer
+          description: Id of the event that the discussion belongs to.
+          example: 12
+        text:
+          type: string
+          description: The text of the post.
+          example: When will the event start, exactly?
+responses:
+  200:
```

For creation status code is 201
```suggestion
  201:
```